.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @paketo-buildpacks/tooling-maintainers
```

Comment:
You'll want to add a `.syncignore` for this as well so that the next PR that updates the `.github` config doesn't wipe it out.

Reply:
Is that necessary? These actions are taken from the `packit` repository which does not get automatically updated so I was not planning on adding automatic updating to this repository.

Reply:
Oh, right. I guess this is fine then.